cluster-autoscaler/FAQ.md

```diff
@@ -590,7 +590,7 @@ any nodes left unregistered after this time.
 Every 10 seconds (configurable by `--scan-interval` flag), if no scale-up is
 needed, Cluster Autoscaler checks which nodes are unneeded. A node is considered for removal when **all** below conditions hold:
 
-* The sum of cpu and memory requests of all pods running on this node ([DaemonSet pods](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/) and [Mirror pods](https://kubernetes.io/docs/tasks/configure-pod-container/static-pod/) are included by default but this is configurable with `--ignore-daemonsets-utilization` and `--ignore-mirror-pods-utilization` flags) is smaller
+* The maximum value between the sum of cpu requests and sum of memory requests of all pods running on this node ([DaemonSet pods](https://kubernetes.io/docs/concepts/workloads/controllers/daemonset/) and [Mirror pods](https://kubernetes.io/docs/tasks/configure-pod-container/static-pod/) are included by default but this is configurable with `--ignore-daemonsets-utilization` and `--ignore-mirror-pods-utilization` flags) is smaller
```

Comment:
Rather than talking about max of sums, wouldn't it be clearer to say that *both* values have to be below the threshold?

Reply:
hi @x13n , it sounds good, i'll merge your contribution as well. Thank you  :)

Reply:
hi @x13n , it sounds good, i'll merge your contribution as well. Thank you  :)

Reply:
Hi @x13n, the change has been applied only in this section. The flag definitions were left as is for the sake of clarity.

Reply:
Thanks @faan11
It looks good to me.
/lgtm

Reply:
removing the `/hold` label as the above suggestion is addressed.

Reply:
/unhold